### PR TITLE
largest-series-product corner cases

### PR DESCRIPTION
**Pull request by petertseng**
As part of the work to have consistency between tracks as in https://github.com/exercism/todo/issues/104 and https://github.com/exercism/todo/issues/13. Comments are in individual commit messages.

### CODE REVIEW

**Comment by kytrinyx** [comment]
@rubysolo or @canweriotnow do you have a moment to take a look at this, please?

**Comment by yurrriq** [comment]
LGTM and the tests pass. :shipit:

PS It seems I'm basically the de facto maintainer of xClojure for now.

File: largest-series-product/test/largest_series_product_test.clj [hunk 1]
```diff
@@ -24,9 +24,11 @@
...
            (lsp/largest-product 6 ds))))
-  (is (thrown-with-msg? Throwable #"empty"
-        (lsp/largest-product 0 "")))
-  ;; unlike the Ruby implementation no error is expected for too small input
-  (is (= 1 (lsp/largest-product 4 "123")))
-  ;; edge case :)
+  ;; edge cases involving 0 or empty strings
+  (is (= 1 (lsp/largest-product 0 "")))
+  (is (= 1 (lsp/largest-product 0 "123")))
+  (is (thrown? Throwable (lsp/largest-product 1 "")))
```
Comment (yurrriq) [inline largest-series-product/test/largest_series_product_test.clj]: :+1: for `s/thrown-with-msg\?/thrown/g`

**Comment by yurrriq** [comment]
Thanks, @peterseng. If you're feeling ambitious and want to address this issue on [xlfe](https://github.com/exercism/xlfe), by all means, go ahead. :smile:

**Comment by kytrinyx** [comment]
@yurrriq Haha, yepp, I think you're right about that.

**Comment by petertseng** [comment]
xlfe is a super interesting case since it has just three tests right now, none of which are testing the interesting edge cases or zero cases. xerlang is the other track that I haven't whipped into shape either. I will see if I can do enough for them to make it work, but I make no promises, heh.

**Comment by yurrriq** [comment]
I more or less ported xerlang to LFE, so that makes sense. I can probably take a look this weekend. I want to move exercises into a subdirectory and maybe add a few more anyway.

**Comment by petertseng** [comment]
As you probably have seen, I decided I was indeed feeling ambitious.

**Comment by yurrriq** [comment]
:grin: Thanks, I'll check it out.
